### PR TITLE
fix(cash): restore show receipt feature

### LINKED ISSUE #2733: Cash Registry's openReceiptModal() method was removed
It appears that the Cash Registry's `openReceiptModal()` was removed so that a user can not longer bring up a PDF receipt.   I think if we just restore the method, everything should work as planned.

### PR DESCRIPTION
**Pull request by jniles**
This commit fixes the cash payments registry link to receipt in the action dropdown.

Closes https://github.com/IMA-WorldHealth/bhima-2.X/issues/2733

### CODE REVIEW

**Comment by jniles** [comment]
@mbayopanda, can I get a review?

File: client/src/js/components/bhReceipt.js [hunk 1]
```diff
@@ -1,34 +1,34 @@
-var bhReceiptTemplate =
-  '<a ng-click="$ctrl.open()" href="">{{ $ctrl.displayValue }}</a>';
+const bhReceiptTemplate = `
+  <a ng-click="$ctrl.open()" href="">
+    {{ $ctrl.displayValue }}
+  </a>`;
```
Comment (mbayopanda) [inline client/src/js/components/bhReceipt.js:4]: 👍

File: client/src/js/components/bhReceipt.js [hunk 2]
```diff
@@ -1,34 +1,34 @@
...
 
 angular.module('bhima.components')
   .component('bhReceipt', {
     template : bhReceiptTemplate,
     controller  : bhReceiptController,
     bindings    : {
       value : '<',
-      displayValue : '<',
       type : '@',
+      displayValue : '<',
```
Comment (mbayopanda) [inline client/src/js/components/bhReceipt.js:13]: Why is it restored in this PR ? Because in this PR i don't see where it is used.

**Review by mbayopanda — APPROVED** [review]
LGTM 👍

File: client/src/js/components/bhReceipt.js (hunk 2 above)
Comment (jniles) [reply client/src/js/components/bhReceipt.js:13]: It wasn't restored, just moved from the second position to the third.  It is used in the template on line 1.

**Pull request merged**